src/wordpress/config.py

```diff
@@ -120,21 +167,39 @@ def config_infos(self, field=None):
 
     @property
     def db_name(self):
+        """
+        Returns WordPress installation DB Name
+        """
         return self.config_infos(field='DB_NAME')
 
     @property
     def db_host(self):
+        """
+        Returns WordPress installation DB host
+        """
         return self.config_infos(field='DB_HOST')
 
     @property
     def db_user(self):
+        """
+        Returns WordPress installation DB user
+        """
         return self.config_infos(field='DB_USER')
 
     @property
     def db_password(self):
+        """
+        Returns WordPress installation DB password for DB_USER
+        """
         return self.config_infos(field='DB_PASSWORD')
 
     def user_infos(self, username=None):
+        """
+        Returns informations about all or one given user.
```

Comment:
i.e a WPUser instance(s), in a dictionnary where keys are user logins

Reply:
Gemacht !